src/components/Activity/DefaultTabBar.js

```diff
@@ -0,0 +1,125 @@
+const React = require('react');
+const ReactNative = require('react-native');
+const {
+  Dimensions,
+  StyleSheet,
+  Text,
+  View,
+  Animated,
+} = ReactNative;
+const Button = require('./Button');
+
+const DefaultTabBar = React.createClass({
+  propTypes: {
+    goToPage: React.PropTypes.func,
+    activeTab: React.PropTypes.number,
+    containerWidth: React.PropTypes.number,
+    leftOffset: React.PropTypes.number,
+    rightOffset: React.PropTypes.number,
+    tabs: React.PropTypes.array,
+    backgroundColor: React.PropTypes.string,
+    activeTextColor: React.PropTypes.string,
+    inactiveTextColor: React.PropTypes.string,
+    textStyle: Text.propTypes.style,
+    tabStyle: View.propTypes.style,
+    style: View.propTypes.style,
+    renderTab: React.PropTypes.func,
+    underlineStyle: View.propTypes.style,
+  },
+
+  getDefaultProps() {
+    return {
+      activeTextColor: 'navy',
+      inactiveTextColor: 'black',
+      backgroundColor: null,
+    };
+  },
+
+  renderTab(name, page, isTabActive, onPressHandler) {
+    const { activeTextColor, inactiveTextColor, textStyle, } = this.props;
+    const textColor = isTabActive ? activeTextColor : inactiveTextColor;
+    const fontWeight = isTabActive ? 'bold' : 'normal';
+
+    return <Button
+      style={{flex: 1, }}
+      key={name}
+      accessible={true}
+      accessibilityLabel={name}
+      accessibilityTraits='button'
+      onPress={() => onPressHandler(page)}
+    >
+      <View style={[styles.tab, this.props.tabStyle, ]}>
+        <Text style={[{color: textColor, fontWeight, }, textStyle, ]}>
+          {name}
+        </Text>
+      </View>
+    </Button>;
+  },
+
+  render() {
+    const containerWidth = this.props.containerWidth;
+    const numberOfTabs = this.props.tabs.length;
+    const tabUnderlineStyle = {
+      position: 'absolute',
+      width: containerWidth / numberOfTabs,
+      height: 0.5,
+      backgroundColor: 'navy',
+      bottom: -0.5,
+    };
+
+    const tabUnderlineFullWidthStyle = {
+      position: 'absolute',
+      width: WIDTH,
+      height: 0.5,
+      borderColor: '#f0f0f2',
+      borderBottomWidth: 1,
+      bottom: 0,
+      zIndex: -1,
+    };
+
+    const left = this.props.scrollValue.interpolate({
+      inputRange: [0, 1, ],
+      outputRange: [this.props.leftOffset,  (containerWidth / 2) - this.props.rightOffset, ],
+    });
+
+    return (
+      <View>
+        <View style={[
+          styles.tabs,
+          { backgroundColor: this.props.backgroundColor, },
+          this.props.style,
+          ]}>
```

Comment:
11: Expected indentation of 8 spaces but found 10
13: The closing bracket must be aligned with the line containing the opening tag (expected column 9 on the next line)